02 - Variables, types, control flow.ipynb

```diff
@@ -0,0 +1,2206 @@
+    "a, b, c = 1, 2, 3\n",
+    "\n",
+    "print(a or b or c)\n",
+    "print(a and b and c)\n",
+    "print(a or 0 or c)\n",
+    "print(a and 0 and c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `None`\n",
+    "\n",
+    "`None` е стойност (и тип), обозначаваща липса на стойност (🤔).\n",
+    "\n",
+    "Когато една функция **не** преминава през `return` израз, тя всъщност връща `None` като резултат."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The print function isn't intended to return a specific value.\n",
+      "None\n"
+     ]
+    }
+   ],
+   "source": [
+    "result_of_print = print(\"The print function isn't intended to return a specific value.\")\n",
+    "print(result_of_print)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `type`\n",
+    "`type` е функция, която ни връща типа на дадена стойност:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "int"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(42)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Но всичко в Python е обект, включително и функциите:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "builtin_function_or_method"
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(print)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Следователно, `type` също си има тип:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "type"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(type)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "😵‍💫"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "type"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(type(type(type(type(type)))))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `tuple`, `list`, `set`, `dict`\n",
+    "\n",
+    "* `tuple`: наредена $n$-торка (още кортеж или вектор), която не може да се променя \n",
```

Comment:
Not sure about the use of the word "vector" - the first thing that pops to my mind is the C++ vector and not vector in the mathematical sense.